CMakeModules/build_glbinding.cmake

```diff
@@ -2,12 +2,8 @@ INCLUDE(ExternalProject)
 
 SET(prefix ${PROJECT_BINARY_DIR}/third_party/glb)
 
-SET(LIB_POSTFIX "")
-IF (${CMAKE_BUILD_TYPE} STREQUAL "Debug")
-    SET(LIB_POSTFIX "d")
-ENDIF()
-
-SET(glbinding_location ${prefix}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}glbinding${LIB_POSTFIX}${CMAKE_STATIC_LIBRARY_SUFFIX})
+SET(glbinding_location ${prefix}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}glbinding${CMAKE_STATIC_LIBRARY_SUFFIX})
```

Comment:
if you build a solution for VS not seting CMAKE_BUILD_TYPE (so that the build type is select-able in VS) then CMAKE_BUILD_TYPE is empty and this "IF (${CMAKE_BUILD_TYPE} STREQUAL "Debug")" throws and error.